css/_components/avatar/_avatar.css

```diff
@@ -0,0 +1,34 @@
+.avatar {
+  border-radius: var(--br-full);
+  height: 50px;
+  width: 50px;
+}
+
+.av-xs {
+  height: 25px;
+  width: 25px;
+}
+
+.av-sm {
+  height: 40px;
+  width: 40px;
+}
+
+.av-md {
+  height: 50px;
+  width: 50px;
+}
+
+.av-lg {
+  height: 70px;
+  width: 70px;
+}
+
+.av-xl {
+  height: 90px;
+  width: 90px;
+}
+
```

Comment:
if you use a root font size of 62.5% and "rems" , it would help you a lot in making your page responsive.

Reply:
My thought process was that avatars won't need to be responsive and also as I have 6 different sizes of avatars so I didn't feel it necessary to make them responsive.